core/src/main/scala/org/apache/spark/deploy/worker/ui/WorkerPage.scala

```diff
@@ -112,7 +112,15 @@ private[ui] class WorkerPage(parent: WorkerWebUI) extends WebUIPage("") {
       <td>
         <ul class="unstyled">
           <li><strong>ID:</strong> {executor.appId}</li>
-          <li><strong>Name:</strong> {executor.appDesc.name}</li>
+          <li><strong>Name:</strong>
+          {
+            if ({executor.state == ExecutorState.RUNNING}) {
+              <a href={executor.appDesc.appUiUrl}> {executor.appDesc.name}</a>
```

Comment:
1. The jump page is the current page. sounds not a good UX. I can add a non-empty judgment.

2. When Iuse spark-shell, exeutor.appDesc.appUIUrl equal '10.43.183.117:4040'

![image](https://user-images.githubusercontent.com/26266482/27717624-ffacb354-5d78-11e7-971a-2d6f3269a722.png)